person.rb

```diff
@@ -1,14 +1,15 @@
 require './nameable'
 class Person < Nameable
+  attr_reader :rental, :id
```

Comment:
- [ ] Let's kindly remind ourselves that a person should have a name, age and so many rentals. Hence we need a `name`, `age`, and `rentals` getters and setters since they can be mortified that's

```.ruby
attr_accessor :name, :age, :rentals

```